tutorials/get-started-canonical-kubernetes/get-started-canonical-kubernetes.md

```diff
@@ -1,187 +1,179 @@
- 
-1. For **Ubuntu** users, install the [kubectl command line](https://kubernetes.io/docs/tasks/tools/install-kubectl/): 
+
+
+### Operate your cluster with kubectl
+
+To deploy and manage applications on Kubernetes you need *kubectl*. Learn more
+by visiting the [official documentation](https://kubernetes.io/docs/user-guide/kubectl/).
+
+1. For **Ubuntu** users, install the [kubectl command line](https://kubernetes.io/docs/tasks/tools/install-kubectl/):
 `$ sudo snap install kubectl --classic`
 If you are running **macOS**:
 `$ brew install kubectl`
 For **Windows** users, use [Chocolatey](https://chocolatey.org/install):
 `$ choco install kubectl`
 For **CentOS and other Linuxes**:
-`$ sudo snap install kubectl --classic` 
- 
+`$ sudo snap install kubectl --classic`
 2. Verify you can run kubectl. You will see a list of commands to control your cluster.
 `$ kubectl`
- 
-3. To connect to JAAS from the command line you'll need to register with the JAAS controller. You’ll only need to do this the first time.
-`$ juju register jimm.jujucharms.com`
- 
-4. This command will open a new window in your browser. Use Ubuntu SSO to login and authorise your account. 
- 
-5. You will then be asked to enter a descriptive name for the JAAS controller.  We suggest using *jaas*. 
- 
-6. JAAS users with existing models might first need to switch to the relevant model. To see all your models:
-`$ juju models`
-
-7. To select the relevant one, run:
-`$ juju switch <model-name>`
- 
- 
- 
-## Operating your Kubernetes cluster 
+
+
+## Operating your Kubernetes cluster
 Duration: 5:00
- 
-1. Fetch the credentials from the Kubernetes Master charm. 
-`$ mkdir -p ~/.kube`
 
-2. Then run:
+To operate your cluster with kubectl you need to download the configuration
+needed to speak to your cluster. The first step to getting the config is to
+make sure you have an SSH key in your JAAS deployment of your cluster.
+
+1. To connect to JAAS from the command line you'll need to register with the JAAS controller. You’ll only need to do this the first time.
+`$ juju register jimm.jujucharms.com`
+2. This command will open a new window in your browser. Use Ubuntu SSO to login and authorise your account
+3. You will then be asked to enter a descriptive name for the JAAS controller.  We suggest using *jaas*
+4. You can now import you SSH key from Launchpad or Github into your cluster. For instance, a user with the Github username 'ghuser' would use the following command:
+`$ juju import-ssh-key gh:ghuser`
+5. Verify your SSH key is working by attempting to run the date command on the
+   first machine
+`$ juju run --machine 0 -- date
+With SSH operating properly you can now setup *kubectl*. Learn more by visiting the [official documentation](https://kubernetes.io/docs/user-guide/kubectl/)
+6. Fetch the credentials from the Kubernetes Master charm
+`$ mkdir -p ~/.kube`
```

Comment:
Move this into step 7 as "Make a directory: `$ mkdir -p ~/.kube` and then copy the config file `$ juju scp kubernetes-master/0:config ~/.kube/config`